INSTRUCTION.md

```diff
@@ -0,0 +1,19 @@
+# Instructions to Run and Stop the Application
+
+## Prerequisites
+- Docker and Docker Compose must be installed on your system.
```

Comment:
Ensure that the instructions specify the minimum required versions of Docker and Docker Compose, if applicable, to avoid compatibility issues.